eitprocessing/datahandling/breath.py

```diff
@@ -0,0 +1,9 @@
+from typing import NamedTuple
+
+
+class Breath(NamedTuple):
```

Comment:
I'm not sure whether that makes sense to me. Labelling all individual breaths is not very useful. It might be useful to label specific breaths at some point, e.g. to indicate which breath in sequence a corresponds to which breath in sequence b. 

I'm currently working on an algorithm using breaths, and I don't think numbering of sequential breaths is necessary. This also introduces extra complexity. E.g.: do you reset the numbering when slicing? Do you reset the numbering when concatenating? Etc. 

The algorithms that was used can be attached to the IntervalData object that contains these Breaths. The code I'm currently using in my project is below. 

```python
breath_detection_params = {
    "framerate": sequence.eit_data["raw"].framerate,
    "minimum_distance": 0.5 / respiratory_rate,
}
breath_detection = BreathDetection(**breath_detection_params)

breaths = breath_detection.find_breaths(
    peep_seq.continuous_data["global_impedance_(mdn)"].values
)
starts, _, ends = zip(*breaths, strict=True)
peep_seq.interval_data.add(
    IntervalData(
        "detected_breaths",
        "Detected breaths",
        None,
        "breaths",
        parameters={BreathDetection: breath_detection_params},
        time_ranges=list(
            zip(time[list(starts)], time[list(ends)], strict=True)
        ),
        values=breaths,
    )
)
```